ui/app/components/license-banners.js

```diff
@@ -31,8 +31,12 @@ export default class LicenseBanners extends Component {
 
   constructor() {
     super(...arguments);
-    // do not dismiss any banners if the user has updated their version
-    const dismissedBanner = localStorage.getItem(`dismiss-license-banner-${this.currentVersion}`); // returns either warning or expired
+    // do not dismiss any banners if the user has updated their license, which is indicated by a change in the this.args.expiry.
+    // do not dismiss any banners if the user has updated their version.
+    const dismissedBanner = localStorage.getItem(
+      `dismiss-license-banner-${this.currentVersion}-${this.args.expiry}`
+    ); // returns either warning or expired`); // returns either warning or expired
+
```

Comment:
the comment is copied twice here, but i'm still a little confused. is this just returning the banner type of either 'warning' or 'expired'? thoughts on updating to something like: 
```suggestion
// type is either 'warning' or 'expired'
    const bannerType = localStorage.getItem(
      `dismiss-license-banner-${this.currentVersion}-${this.args.expiry}`
    ); 

```

Reply:
This has 3 comments in a row with this style, so I'm going to keep it after the function. It's hard to see that in the git diff view. And yes, thank you for the catch on the copied twice.